dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -149,6 +70,9 @@ const LastUpdated = ({
 	);
 };
 
+// TODO: update this code to use shared version when it is available
```

Comment:
Random aside: I have about a 20% hit rate on actually going back and really doing those things I write todos for - I have all the right intentions, it's just other stuff comes along. The rest just sit in the code and linger. There are currently 97 TODOs in DCR, I'm sure many of them are mine.

I wonder if rather than writing them as notes to self, if maybe we should write them as messages to future developers, saying that there are some improvements they can help with. With that mindset we would add hints and tips to help these future versions of us and the task would stand a much better chance of being done

Reply:
Yes, perhaps we should. There is a trello card to get this work done, so perhaps that is a more reliable way of ensuring that this particular TODO doesn't end up being another piece of code flotsam.